dtfs-central-api/src/helpers/fee-record-matching.test.ts

```diff
@@ -615,12 +634,9 @@ describe('fee-record-matching', () => {
 
             const payments = [PaymentEntityMockBuilder.forCurrency(paymentCurrency).withAmount(paymentAmount).build()];
 
-            jest.mocked(getActivePaymentMatchingTolerances).mockResolvedValue({
-              USD: 0,
-              JPY: 0,
-              EUR: 0,
-              GBP: tolerance,
-            });
+            mockFindOneByCurrencyAndIsActiveTrue.mockResolvedValue(
```

Comment:
Just out of curiosity, do you think there is a benefit to using `jest-when` here? I haven't been able to draw a line in my head yet where we should use `jest-when` and where it is excessive, so I'm curious to hear your thoughts

Reply:
So I don't think it really matters? I think my opinion is that if I was asserting the call in every test then it would be better to use `jest-when`, instead to remove all those additional assertions, instead I have one test that specifically fails if we call it with the wrong parameters and then the others will only fail for separate reasons which I like since it aligns with the test only failing for one reason (or as little reasons as possible) which we strive for in unit tests.

Reply:
That sounds along the same lines of thinking I have, so sounds good to me!